### PR TITLE
Create new.yaml

### PR DESCRIPTION
**Pull request by Amitbergman**
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> After the submission has been made, please look at the Validation Checks.

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Detections/ASimAuthentication/new.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+tags:
+  - Id: 28b42356-45af-40a6-a0b4-a554cdfd5d8a
```
Comment (shainw) [inline Detections/ASimAuthentication/new.yaml:20]: What is this Id for?

File: Detections/ASimAuthentication/new.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+severity: Medium
+requiredDataConnectors: []
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
```
Comment (shainw) [inline Detections/ASimAuthentication/new.yaml:18]: Can you include the sub-technique?

**Review by shainw — CHANGES_REQUESTED** [review]
Couple of comments -

- The name of the YAML file needs to be fixed. 
- Are we looking at an official connector reference for Windows Events?
- We have several queries already covering 4624, it seems like this could be integrated into one of those.  Can you have a look please - https://github.com/Azure/Azure-Sentinel/search?l=YAML&q=4624

**Comment by oshezaf** [comment]
@shainw : I suspect this is a test PR. Amit is not creating ASIM detections (at least yet). @Amitbergman : am I correct here?

**Comment by vmanojreddy** [comment]
@Amitbergman please update the requested changes ,thanks!!!

**Comment by vmanojreddy** [comment]
@Amitbergman please update the requested changes ,thanks!!!

**Comment by vmanojreddy** [comment]
@Amitbergman please update the requested changes ,thanks!!!

**Comment by aprakash13** [comment]
Reached out to Amit to follow up on this.

**Pull request closed without merging**